irohad/model/commands/add_signatory.hpp

```diff
@@ -40,7 +40,6 @@ namespace iroha {
       pubkey_t pubkey;
 
       bool operator==(const Command &command) const override;
-      bool operator!=(const Command &command) const override;
```

Comment:
Why in this command you delete it, and others not

Reply:
This operators are redundant, because they are implemented in command.cpp. I didn't do those changes, they were done by luckychess. Don't understand how this commit happened in my branch:)

Reply:
Got it, please rebase with some picks/ignore then